css/style.css

```diff
@@ -349,15 +328,15 @@ pre code {
 #top {
   line-height: 0;
   background: black;
-  -webkit-border-radius: 2px;
+  border-radius: 2px;
   position: fixed;
   bottom: 15px;
   right: 15px;
   padding: 8px;
   text-decoration: none;
   color: white;
   opacity: 0;
-  -webkit-transition: opacity 300ms;

```

Comment:
Keep it and also keep your change too.

Reply:
This property has good support, just like the previous two. I think it would be better to remove the prefix. [ref](https://caniuse.com/mdn-css_properties_transition)

Reply:
Yeah okay.